src/components/PlanningTable/index.tsx

```diff
@@ -14,30 +14,22 @@ import {
   useReactTable
 } from '@tanstack/react-table'
 
-import {
-  Table, TableBody, TableCell, TableRow
-} from '@ttab/elephant-ui'
-
+import { Table, TableBody, TableCell, TableRow } from '@ttab/elephant-ui'
 import { Pagination } from './Pagination'
 import { Toolbar } from './Toolbar'
 
 interface PlanningTableProps<TData, TValue> {
   columns: Array<ColumnDef<TData, TValue>>
   data: TData[]
+  onRowSelected?: (row?: TData) => void
 }
 
 
-export const PlanningTable = <TData, TValue>({
-  columns,
-  data
-}: PlanningTableProps<TData, TValue>): JSX.Element => {
-  const [rowSelection, setRowSelection] = React.useState({})
-  const [columnVisibility, setColumnVisibility] =
-    React.useState<VisibilityState>({})
-  const [columnFilters, setColumnFilters] = React.useState<ColumnFiltersState>(
-    []
-  )
-  const [sorting, setSorting] = React.useState<SortingState>([])
+export const PlanningTable = <TData, TValue>({ columns, data, onRowSelected }: PlanningTableProps<TData, TValue>): JSX.Element => {
```

Comment:
I suppose this now a one liner for readability, and with that I some what agree. Unfortunately I think this decreases the visibility, in my opinion 80-100 chars is the max length for a line.